apps/wallet-mobile/src/features/Discover/useDappConnectorManager.ts

```diff
@@ -31,69 +29,84 @@ export const useDappConnectorManager = () => {
         appStorage,
         wallet,
         confirmConnection,
-        signTx,
+        signTx: (cbor) => {
```

Comment:
I'd expect the API to be `signTx` only, and then pass the params like isHW, or whatever is needed to decide which path should it take.